### PR TITLE
Get first CWL step(s) working

### PR DESCRIPTION
**Pull request by gijzelaerr**

### CODE REVIEW

**Comment by tammojan** [comment]
Please keep `setup_logger`, it's used by commissioners now.

File: apercal/modules/preflag.py [hunk 1]
```diff
@@ -870,31 +914,33 @@ def manualflag_time(self):
...
                     logger.debug('Flagging time range ' + self.preflag_manualflag_time + ' for all target beams')
                 else:
                     beams = self.preflag_manualflag_targetbeams.split(",")
-                    datasets = [self.basedir + str(b).zfill(2) + '/' + self.rawsubdir + '/' + self.target for b in beams]
+                    datasets = [self.get_datasets_beams(beams)]
                     logger.debug('Flagging time range ' + self.preflag_manualflag_time + ' for selected target beams')
                 for vis in datasets:
-                    if preflagtargetbeamsmanualflagtime[int(vis.split('/')[-3])] == self.preflag_manualflag_time:
-                        logger.info('Time range ' + self.preflag_manualflag_time + ' for target beam ' + vis.split('/')[-3] + ' was already flagged')
+                    band = vis.split('/')[-3]
```
Comment (tammojan) [inline apercal/modules/preflag.py]: Please rename band to beam_nr here

File: apercal/modules/preflag.py [hunk 2]
```diff
@@ -951,97 +1001,127 @@ def aoflagger_flag(self):
...
-        preflagaoflaggertargetbeamsflag = get_param_def(self, 'preflag_aoflagger_targetbeams_flag_status', np.full((beams), False))  # AOFlagged the target beams?
+        # AOFlagged the flux calibrator?
+        preflagaoflaggerfluxcalflag = get_param_def(self, 'preflag_aoflagger_fluxcal_flag_status', False)
+
+        # AOFlagged the polarised calibrator?
+        preflagaoflaggerpolcalflag = get_param_def(self, 'preflag_aoflagger_polcal_flag_status', False)
+
+        # AOFlagged the target beams?
+        preflagaoflaggertargetbeamsflag = get_param_def(self, 'preflag_aoflagger_targetbeams_flag_status',
+                                                        np.full((beams), False))
```
Comment (tammojan) [inline apercal/modules/preflag.py]: If you want this code change (comment before the line), please use a regexp on all files; this construct is all over the place.

File: apercal/modules/preflag.py (hunk 1 above)
Comment (gijzelaerr) [reply apercal/modules/preflag.py]: yeah already done

File: apercal/modules/default.cfg [hunk 3]
```diff
@@ -14,6 +14,7 @@ contsubdir = 'continuum'                            # Sub-directory for continuu
 polsubdir = 'polarisation'                          # Sub-directory for polarisation imaging, e.g. 'polarisation'
 mossubdir = 'mosaics'                               # Sub-directory for masaicking, e.g. 'mosaics'
 transfersubdir = 'transfer'                         # Sub-directory for the transfer of the final (u,v)-datasets, e.g. 'transfer'
+subdirification = True                              # assume data is in /basedir/bandnum/rawsubdir/fluxcal format
```
Comment (tammojan) [inline apercal/modules/default.cfg]: Comment should also read /basedir/beamnum/rawsubdir/fluxcal

File: apercal/modules/preflag.py (hunk 2 above)
Comment (gijzelaerr) [reply apercal/modules/preflag.py]: i couldnt' find a way to do this

**Comment by tammojan** [comment]
Please set log level to debug in tests.

**Pull request merged**